### PR TITLE
fix: remove useless window interface

### PR DESCRIPTION
**Pull request by TekMath**
## Changes made

Removed useless class because path of the window icon is already in Window props and we don't need to load something that represent an window icon before the initialization of this.

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .vscode/
+.idea/
```
Comment (flavien-chenu) [inline .gitignore]: Missing last line

**Review by flavien-chenu — CHANGES_REQUESTED** [review]

**Review by flavien-chenu — COMMENTED** [review]
200 OK. ploplop

**Review by Yann-Masson — APPROVED** [review]
.002 KO

**Review by flavien-chenu — APPROVED** [review]
200 OK (loosed packets). GG TekMath trop bg (seulement le jeudi)

**Review by sdragos1 — APPROVED** [review]

**Pull request merged**